android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
What kind of issues ?
In your module you don't convert you filePath into data ?

Every upload provider now have a rest API. You can upload from RN without any problem (never face one)
[Amazon S3 Browser-Based Upload using HTTP POST](http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-post-example.html)

There is an exemple (with cloudinary rest API)
https://github.com/marcshilling/react-native-image-picker/issues/61#issuecomment-164467359